.github/workflows/containers-apply.yml

```
@@ -0,0 +1,69 @@
+
+name: Container plan
+on:
+  workflow-dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - "containers/**"
+      - ".github/workflows/containers-apply.yml"
+
+env: 
+  - AWS_REGION: ca-central-1
+  - REGISTRY: 066023111852.dkr.ecr.ca-central-1.amazonaws.com/superset
+  - GITHUB_SHA: ${{ github.sha }}
```

Does this work with the array notation?  I've only seen it like so:
```suggestion
  AWS_REGION: ca-central-1
  REGISTRY: 066023111852.dkr.ecr.ca-central-1.amazonaws.com/superset
  GITHUB_SHA: ${{ github.sha }}
```

---

probably not, it's been a bit since I wrote a new action